esp-hal/CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   MCPWM clusters (#1360)
   TIMG clusters (#1364)
   SOC_ETM clusters (#1365)
+  LEDC clusters (#1368)
```

Comment:
What is the reason for adding `CHANGELOG` entries for all of these PAC changes? This has absolutely zero impact on the end user as far as I can tell, and just appears to be implementation details. This just adds visual noise and does not present any useful information, IMO.

Reply:
Fair point, I will remove them in my other PR and from now on add skip-changelog.